src-electron/generator/matter/controller/python/templates/helper.js

```diff
@@ -43,6 +49,60 @@ function asPythonType(zclType) {
   }
 }
 
+/**
+ *
+ * Available options:
+ * - All options passed to this helper are considered as overrides for
+ * zcl types
+ * for eg: (as_underlying_python_zcl_type type clusterId boolean='bool')
+ * will return "bool" for "boolean" type
+ * @param {*} type
+ * @param {*} clusterId
+ * @param {*} options
+ * @returns The corresponding python data type for a zcl data type.
+ */
+async function as_underlying_python_zcl_type(type, clusterId, options) {
+  let hash = options.hash;
+  // Overwrite any type with the one coming from the template options
+  // Eg: {{as_underlying_python_zcl_type type [clusterId] boolean='bool'}}
```

Comment:
Maybe I was unclear:

I am ok leaving the override functionality if we think it might be useful in the future, but we should doing the right thing for booleans without any overrides.

Reply:
ok I can change the templates for this accordingly.

Reply:
That's fine, but I'm not sure why we're not just making this helper do the right thing, as long as we have this helper....